README.md

```diff
@@ -0,0 +1,141 @@
+something new while taking care of daily tasks. These can be:
+
+* handling releases or deployments
+* running meetings
+* grooming backlog
+* keeping production service alive
+* interacting with other teams, users and community
+
+These should be a responsibility of the whole team, not just a single person.
+With rotating roles, you can have an engaged team that takes turns in the daily
+operations and is not disrupted when a key member leaves for a vacation.
+
+This repository is an evolution of the rotating roles that we started in the Packit team. You can see it in action [here](https://github.com/packit/agile) and check role assignments in the [repository's issues](https://github.com/packit/agile/issues).
+
+After the technique proved successful, we followed with a series of blog posts and presentations:
+1. 📄 [How to run an open-source service?](https://medium.com/@laura.barcziova/how-to-run-an-open-source-service-fb3303240e69)
+2. 📄 [Inception of rotating roles](https://medium.com/@laura.barcziova/inception-of-rotating-roles-9caf971b3096)
+3. 📄 [Share team responsibilities efficiently](https://medium.com/@laura.barcziova/share-team-responsibilities-efficiently-9a202aad7bd0)
+4. 📄 [Role rotation tutorial](https://medium.com/@laura.barcziova/role-rotation-tutorial-957ed3545ef2)
+5. 📽️ [A journey of iteration and role rotation for better collaboration - DevConf.CZ 2024](https://www.youtube.com/watch?v=y1t7Wd31bL8)
+
+
+## Common team roles
+We have interviewed several colleagues of ours from Red Hat. The conclusion we
+reached was there are multiple common roles that apply in many teams. We will
+define them here so you can easily reuse the roles in your team. They don’t
+apply everywhere and to everyone —  you are welcome to be inspired and create a
+role that will fit your team.
+
+## 🗓️Meeting lead
+Having a productive meeting that sticks to agenda, has notes, and a clear
+outcome, has a tremendeous impact on everyone. With one responsible meeting
+lead, you can stop wasting time of all people on your meetings.
+
+### Responsible for
+* Run a meeting (TBD: this could be more detailed)
+* Make sure people stay on topic
+* Follow meeting agenda - address all points
+* Write down action items and assign them
+* Make meeting notes or assign someone on the call to write them down
+
+### Why have this role?
+* Meetings need moderators, otherwise they succumb to chaos
+* Well run meeting is the difference between a complete waste of time and
+  effectively coordinating in a group of people
+
+### What can people learn?
+* Not everyone likes the attention, so this can be a big step outside of
+  comfort zone for many.
+* It can be tiring to be in charge of everything all the time. Managers
+  understand. Team members may suddenly feel more empathy and compassion once
+  they realize that leading a meeting requires significant effort.
+* Better knowledge about the code base to be able to review the changes.
```

Comment:
```suggestion

```
this is probably by mistake taken from the review role

Reply:
oh my, you are right, I placed it incorrectly; thank you!